src/chainparamsbase.cpp

```diff
@@ -33,7 +33,7 @@ class CBaseMainParams : public CBaseChainParams
 public:
     CBaseMainParams()
     {
-        nRPCPort = 10889;
```

Comment:
Please revert this

Reply:
Presently both **nRPCPort** and **nDefaultPort** are configured to use 10889.  

Changing nRPCPort doesn't affect the P2P network and anybody already using the RPC service is already redefining it either in the conf file or via command line.  This change would be transparent to end users.